community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/LoadCsvAcceptanceTest.scala

```diff
@@ -381,6 +381,20 @@ class LoadCsvAcceptanceTest
     result.toList should equal(List(Map("field" -> "something")))
   }
 
+  test("should restrict file urls to be rooted within an authorized directory") {
+    val dir = createTempDirectory("loadcsvroot")
+
+    val db = new TestGraphDatabaseFactory()
```

Comment:
I'm not following here. The test IS using `execute`, and how are there two databases running?

It's basically a copy paste of the test immediately above: `should allow paths relative to authorized directory`

@pontusmelke  Could you show me how it should be written to avoid the dual databases?